manim/scene/scene_file_writer.py

```diff
@@ -73,7 +73,8 @@ def __init__(self, renderer, scene_name, **kwargs):
         self.partial_movie_files: List[str] = []
         self.sections: List[Section] = []
         # first section gets automatically created for convenience
-        self.next_section("autocreated", DefaultSectionType.NORMAL)
+        # if you need the first section to be skipped, add a first section by hand, it will replace this one
+        self.next_section("autocreated", DefaultSectionType.NORMAL, False)
```

Comment:
For the sake of readability I'd prefer if these were passed as kwargs.

```suggestion
        self.next_section(name="autocreated", type=DefaultSectionType.NORMAL, skip_animations=False)
```

Reply:
Makes sense.
One thing I really don't like about this is that both type and name are strings. I've already received complaints about users switching them up, Manim not caring about that and the Manim Editor throwing errors apparently out of nowhere. But I really don't know how to implement an extensible type system that isn't using strings.

Reply:
I think it is easier to consequently use kwargs for specifying type and skip_animations; we could require that the type has to be passed via a keyword argument, this might reduce confusion about which parameter is what. 🤔